Makefile

```diff
@@ -7,6 +7,7 @@ dist: $(DISTFILE)
 install:
 	@rm -rf $(DESTDIR)
 	./tools/install.py $(DESTDIR)
+	npm install --prefix $(DESTDIR)
```

Comment:
Is that the same as doing `cd $(DESTDIR) && npm install`?

Reply:
Yes.